policyengine_uk/variables/gov/dwp/LHA_category.py

```diff
@@ -36,11 +36,14 @@ def formula(benunit, period, parameters):
             household.max(person("age", period))
         )
         has_children = benunit.any(person("is_child", period))
-        # Households with only one adult, if under 35, can only claim shared if without children:
+        # Households with only one adult, if under age threshold, can only claim shared if without children:
         # https://www.legislation.gov.uk/uksi/2013/376/schedule/4/paragraph/28
+        age_threshold = parameters(
```

Comment:
Use `p.` to indicate it's a parameter
```suggestion
        p = parameters(period).gov.dwp.LHA
```